tests/spicy/types/bitfield/parse-bitorder-2.spicy

```diff
@@ -0,0 +1,143 @@
+# @TEST-EXEC: echo "" | spicy-driver %INPUT >output
```

Comment:
Nit here: better not to use `-<N>` naming on test files. `btest` creates those internally when using `@BTEST-START-NEXT`. Using them for both cases can be confusing, also if one might want to add a `@BTEST-START-NEXT` to an existing test.

Reply:
Ah, yes! parse-bitorder-endianness.spicy it has become.